packages/remote-config/test/remote_config.test.ts

```diff
@@ -390,39 +390,58 @@ describe('RemoteConfig', () => {
     const ETAG = 'etag';
     const CONFIG = { key: 'val' };
     const NEW_ETAG = 'new_etag';
+    const TEMPLATE_VERSION = 1;
 
     let getLastSuccessfulFetchResponseStub: sinon.SinonStub;
     let getActiveConfigEtagStub: sinon.SinonStub;
+    let getActiveConfigTemplateVersionStub: sinon.SinonStub;
     let setActiveConfigEtagStub: sinon.SinonStub;
     let setActiveConfigStub: sinon.SinonStub;
+    let setActiveConfigTemplateVersionStub: sinon.SinonStub;
 
     beforeEach(() => {
       getLastSuccessfulFetchResponseStub = sinon.stub();
       getActiveConfigEtagStub = sinon.stub();
+      getActiveConfigTemplateVersionStub = sinon.stub();
       setActiveConfigEtagStub = sinon.stub();
       setActiveConfigStub = sinon.stub();
+      setActiveConfigTemplateVersionStub = sinon.stub();
 
       storage.getLastSuccessfulFetchResponse =
         getLastSuccessfulFetchResponseStub;
       storage.getActiveConfigEtag = getActiveConfigEtagStub;
+      storage.getActiveConfigTemplateVersion =
+        getActiveConfigTemplateVersionStub;
       storage.setActiveConfigEtag = setActiveConfigEtagStub;
       storageCache.setActiveConfig = setActiveConfigStub;
+      storage.setActiveConfigTemplateVersion =
+        setActiveConfigTemplateVersionStub;
     });
 
     it('does not activate if last successful fetch response is undefined', async () => {
       getLastSuccessfulFetchResponseStub.returns(Promise.resolve());
       getActiveConfigEtagStub.returns(Promise.resolve(ETAG));
+      getActiveConfigTemplateVersionStub.returns(
+        Promise.resolve(TEMPLATE_VERSION)
+      );
 
       const activateResponse = await activate(rc);
 
       expect(activateResponse).to.be.false;
       expect(storage.setActiveConfigEtag).to.not.have.been.called;
       expect(storageCache.setActiveConfig).to.not.have.been.called;
+      expect(
+        storage.setActiveConfigTemplateVersion
+      ).to.not.have.been.calledWith(TEMPLATE_VERSION);
```

Comment:
If I understand the purpose of this test, this should be simply `not.have.been.called`, as you want the method to not be called at all, not just not called with a specific argument. Same in line 455.